src/components/ScriptExplorer/SignatureImporter.jsx

```diff
@@ -36,7 +36,7 @@ import {
   setSignatureImporterFinalized,
   setSignatureImporterComplete,
 } from "../../actions/signatureImporterActions";
-import "react-table/react-table.css";
+// import "react-table-6/react-table.css";
```

Comment:
Whatever CSS we end up using here may as well just be imported at a top-level, right? I think it's included multiple times if it's imported on multiple pages.

Reply:
Correct. But I think I'm going to need to upgrade react-table to the latest major version due peer dependencies so I may end up just deleting this import altogether.